policy/modules/system/init.te

```diff
@@ -37,6 +37,13 @@ gen_tunable(init_daemons_use_tty, false)
 ## </desc>
 gen_tunable(init_mounton_non_security, false)
 
+## <desc>
+## <p>
+## Enable systemd to mount on the SELinux config directories.
```

Comment:
What does kanidm do here?

Reply:
This is to support `kanidm-unixd`'s SELinux feature. When a Kanidm user logs in, their home directory is created in `/home/<UUID>` and a symlink with their username pointing to it at `/home/<username>`. The SELinux feature creates an equivalence rule in the policy so that `/home/<username>` equals `/home/<UUID>`, but the policy needs to be writable.

This tunable allows an administrator to add `ReadWritePaths=/etc/selinux` to the systemd unit which by default has `ProtectSystem=strict` set. Then, `init` will create a read-writable bind mount in the service's mount namespace to support this.

Reply:
To clarify, this tunable here is generic. In the context of Kanidm one would need to enable both this (if using systemd and upstream's systemd unit) and `kanidm_unixd_modify_policy` for it to work.